install/installConfig.php

```diff
@@ -1258,10 +1192,58 @@ private function getFormItems(
 </table>
 </div>
 EOQ;
+        $out .= "</div>";
+
+        // --------------------------
+        //  Advanced Database Configuration
+        // --------------------------------->
 
-        $out.= "</div>";
+        require_once('install/suite_install/collations.php');
 
+        $collationCB = "<select name='setup_db_collation' id='setup_db_collation' class='select' onChange='document.getElementById(\"setup_db_charset\").value = document.getElementById(\"setup_db_collation\").value.split(\"_\")[0];'>";
```

Comment:
Ideally, all of this should be moved into a template but this should be fine for a quick fix.